src/main/java/eu/ai4work/sws/model/generic/SlidingDecisionRequest.java

```diff
@@ -0,0 +1,11 @@
+package eu.ai4work.sws.model.generic;
+
+import lombok.Data;
+import java.util.Map;
+
+@Data
+public class SlidingDecisionRequest {
+    private String decisionStatus;
+    private Map<String, Object> inputParameters;
```

Comment:
When removing the `decisionStatus`, maybe we can make this one more clear by using:
```suggestion
    private Map<String, Object> slidingDecisionInputParameters;
```